scripts/check_docs.py

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env python
+# pylint: disable=invalid-name,redefined-outer-name
+from glob import glob
+import os
+import re
+import sys
+from typing import Set
+
+DOCS_DIR = 'doc/api'
+MODULE_REGEX = r"\ballennlp\.[a-z0-9_.]+\b"
+MODULE_GLOB = 'allennlp/**/*.py'
+
+MODULES_THAT_NEED_NO_DOCS: Set[str] = {
+        # no docs at top level
+        'allennlp',
+
+        # no docs for custom extensions, which aren't even in python
+        'allennlp.custom_extensions',
+        'allennlp.custom_extensions._ext',
+        'allennlp.custom_extensions._ext.highway_lstm_layer',
+        'allennlp.custom_extensions.build',
+
+        # TODO(joelgrus) figure out how to make these docs build, the cffi part
```

Comment:
Missing colon, and it'd be better if all comments here were capitalized and had periods.